src/xtopcom/xtxpool_service_v2/src/xcons_utl.cpp

```diff
@@ -1,66 +1,66 @@
-// Copyright (c) 2017-2020 Telos Foundation & contributors
-// Distributed under the MIT software license, see the accompanying
-// file COPYING or http://www.opensource.org/licenses/mit-license.php.
```

Comment:
maybe file format changed

Reply:
will pay attention to this phenomenon later